### PR TITLE
Back 63

### PR DESCRIPTION
**Pull request by Manizha-khorram**
Sending the message to the user.

Note : First connect users A and B to webSocket and make them listen to the newMessage event then create and send message to the user B. Check your user B webSocket request, should received the message.

### CODE REVIEW

File: src/socket.js [hunk 1]
```diff
@@ -1,43 +1,76 @@
 const socketio = require('socket.io')
 const authMiddleware = require('./../middleware/authenticationSocket')
-const { listPartnerUsers, userTypingStatus } = require('./controllers/message')
+const {
+    listPartnerUsers,
+    userTypingStatus,
+    createMessage,
+} = require('./controllers/message')
+const { default: mongoose } = require('mongoose')
```
Comment (larasodjati) [inline src/socket.js:8]: Nice job Manizha! could you please let me know why do we need mongoose in socket.js?

**Comment by larasodjati** [comment]
I tested it on Postman, and I believe the code working for sending the message, @jpremo could you please review it as well? Thank you!
image

**Review by jpremo — CHANGES_REQUESTED** [review]
I think this change breaks the POST /messages route. I looked into what I mentioned last night and it does seem possible to emit socketio events in response to HTTP requests. Essentially the goal would be to upon receiving a request at POST /messages send the newly created message to receivedByUser's socket (assuming it exists).

The tricky part of this is getting access to the io object with our current app/server setup. I created a basic example [here](https://github.com/Code-the-Dream-School/bb-practicum-team-1-back/tree/BACK-63-alt-example) that shows one way to accomplish this. I'm sure there are different ways to do the same thing that might be a bit more organized.

Let me know if you have questions or if these requests don't make sense.

**Comment by larasodjati** [comment]
You did an excellent job with the changes Manizha! I learned a lot from this PR. 
Hi @jpremo, we need your review and approval on this. Thank you!

**Review by larasodjati — APPROVED** [review]

**Review by jpremo — APPROVED** [review]

**Pull request merged**